io/Cargo.toml

```
@@ -6,4 +6,5 @@ license = "MIT"
 authors = ["Gear Technologies"]
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
+gstd = { git = "https://github.com/gear-tech/gear.git" }
+gmeta = { git = "https://github.com/gear-tech/gear", features = ["codegen"] }
```

```suggestion
gmeta = { git = "https://github.com/gear-tech/gear" }
```
This feature makes sense only in the "state" crate because the feature enables the `#[metawasm]` macro.